data/sea/52-psv.h

```diff
@@ -266,6 +266,7 @@ static ierror_msg_t INLINE psv_output_char
     return 0;
 }
 
+/* Assuming value_size does not include the null terminator */
```

Comment:
@jystic when outputting `chord_time`, I ended up calling this with `chord_size - 1` (see below), hope that always works